src/index.ts

```diff
@@ -1,6 +1,23 @@
-import type { AxiosError, AxiosRequestConfig, AxiosInstance, AxiosStatic } from 'axios';
+import type {
+  AxiosError,
+  AxiosRequestConfig,
+  AxiosInstance,
+  AxiosStatic,
+  AxiosInterceptorManager,
+  AxiosResponse,
+  InternalAxiosRequestConfig
+} from 'axios';
 import isRetryAllowed from 'is-retry-allowed';
 
+interface AxiosResponseInterceptorManagerExtended extends AxiosInterceptorManager<AxiosResponse> {
```

Comment:
There is a `handlers` property in the axios request/response interceptor, but it is not present in the current axios type definition. Therefore, we are extending it on our own.

However, if the following PRs are merged, this definition will no longer be necessary.
https://github.com/axios/axios/pull/6138